--- squad_prepro.py	2020-03-20 09:59:40.779196537 -0300
+++ squad_prepro_new.py	2020-03-20 10:02:08.130342990 -0300
@@ -67,19 +67,12 @@
 
     squad_train_path = os.path.join(root, 'squad/train.json')
     squad_dev_path = os.path.join(root, 'squad/dev.json')
-    squad_v2_train_path = os.path.join(root, 'squad_v2/train.json')
-    squad_v2_dev_path = os.path.join(root, 'squad_v2/dev.json')
-
+    
     squad_train_data = load_data(squad_train_path)
     squad_dev_data = load_data(squad_dev_path, is_train=False)
     logger.info('Loaded {} squad train samples'.format(len(squad_train_data)))
     logger.info('Loaded {} squad dev samples'.format(len(squad_dev_data)))
 
-    squad_v2_train_data = load_data(squad_v2_train_path, v2_on=True)
-    squad_v2_dev_data = load_data(squad_v2_dev_path, is_train=False, v2_on=True)
-    logger.info('Loaded {} squad_v2 train samples'.format(len(squad_v2_train_data)))
-    logger.info('Loaded {} squad_v2 dev samples'.format(len(squad_v2_dev_data)))
-
     canonical_data_suffix = "canonical_data"
     canonical_data_root = os.path.join(root, canonical_data_suffix)
     if not os.path.isdir(canonical_data_root):
@@ -91,13 +84,6 @@
     dump_rows(squad_dev_data, squad_dev_fout, DataFormat.PremiseAndOneHypothesis)
     logger.info('done with squad')
 
-    squad_v2_train_fout = os.path.join(canonical_data_root, 'squad-v2_train.tsv')
-    squad_v2_dev_fout = os.path.join(canonical_data_root, 'squad-v2_dev.tsv')
-    dump_rows(squad_v2_train_data, squad_v2_train_fout, DataFormat.PremiseAndOneHypothesis)
-    dump_rows(squad_v2_dev_data, squad_v2_dev_fout, DataFormat.PremiseAndOneHypothesis)
-    logger.info('done with squad_v2')
-
-
 
 if __name__ == '__main__':
     args = parse_args()
