--- metrics_old.py	2020-03-06 19:38:55.434809370 -0300
+++ metrics.py	2020-03-06 19:39:42.113951185 -0300
@@ -3,6 +3,7 @@
 
 from sklearn.metrics import matthews_corrcoef
 from sklearn.metrics import accuracy_score, f1_score
+from sklearn.metrics import recall_score
 from sklearn.metrics import roc_auc_score
 from scipy.stats import pearsonr, spearmanr
 from seqeval.metrics import classification_report
@@ -14,6 +15,9 @@
 def compute_f1(predicts, labels):
     return 100.0 * f1_score(labels, predicts)
 
+def compute_recall(predicts, labels):
+    return 100.0 * recall_score(labels, predicts)
+
 def compute_mcc(predicts, labels):
     return 100.0 * matthews_corrcoef(labels, predicts)
 
