--- predict.py	2020-02-26 10:20:33.961203000 -0300
+++ predict_new.py	2020-03-10 18:32:56.845255246 -0300
@@ -7,6 +7,8 @@
 from experiments.exp_def import TaskDefs, EncoderModelType
 #from experiments.glue.glue_utils import eval_model
 
+from torch.utils.data import DataLoader
+
 from mt_dnn.batcher import SingleTaskDataset, Collater
 from mt_dnn.model import MTDNNModel
 from data_utils.metrics import calc_metrics
@@ -58,8 +60,7 @@
 encoder_type = config.get('encoder_type', EncoderModelType.BERT)
 # load data
 test_data_set = SingleTaskDataset(args.prep_input, False, task_type=task_type, maxlen=args.max_seq_len)
-collater = Collater(gpu=args.cuda, is_train=False, task_id=args.task_id, task_type=task_type,
-                    data_type=data_type, encoder_type=encoder_type)
+collater = Collater(is_train=False, encoder_type=encoder_type)
 test_data = DataLoader(test_data_set, batch_size=args.batch_size_eval, collate_fn=collater.collate_fn, pin_memory=args.cuda)
 
 with torch.no_grad():
